ocfweb/static/js/pages/account/quick_username_validation.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+    var finTypingCountdown = 250; // 250 milliseconds
+    var typingTimer;
+    var $input = $('#id_ocf_login_name');
+
+    // On keyup, start countdown
+    $input.keyup(function() {
+        //alert('key UP');
+        clearTimeout(typingTimer);
+        typingTimer = setTimeout(validate_username, finTypingCountdown);
+    });
+
+    // On keydown, clear countdown
+    $input.keydown(function() {
+        //alert('key DOWN');
+        clearTimeout(typingTimer);
+    });
+
+    function validate_username() {
+        $.ajax({
+             type: 'GET',
+             url: 'validate',
+             data: {'username': $input.val(),
+                    'real_name': $('#real_name').html()},
+             success: function(data) {
+                 if(data.is_valid) {
+                     $input.css('border', '5px solid #3c763d');
```

Comment:
i think a slightly nicer approach would be to add css classes like `.valid` and `.invalid` and then use `$input.addClass`

in fact, what if you used the bootstrap validation css? https://getbootstrap.com/docs/3.3/css/#forms-control-validation

this css should be on the page, you'd just need to add the right css class names (i think `has-error` and `has-success`)